storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2921,10 +2852,10 @@ static inline void rocksdb_register_tx(handlerton *const hton, THD *const thd,
                                        Rdb_transaction *const tx) {
   DBUG_ASSERT(tx != nullptr);
 
-  trans_register_ha(thd, FALSE, rocksdb_hton, NULL); // TODO: PFS
+  trans_register_ha(thd, FALSE, rocksdb_hton, nullptr);
```

Comment:
Good call, fixing the NULL is enough to warrant fixing the bool at the same time.